.vscode/settings.json

```diff
@@ -43,5 +43,14 @@
     "**/lib": true,
     "**/dist": true
   },
-  "formatFiles.excludePattern": "**/*.min.js"
+  "formatFiles.excludePattern": "**/*.min.js",
+  "prettier.eslintIntegration": true,
+  "eslint.codeAction.showDocumentation": {
+
+    "enable": true
+  },
+  "eslint.nodeEnv": "",
+  "eslint.options": {
+  
+  }
```

Comment:
What is the use of this settings block?

Reply:
I am not sure. I didn't realize that my local vscode settings would be included in a commit. I am not sure how to fix this. I am willing to accept any pull that includes modifications to vscode settings.